### DISCUSSION TITLE
consider the possibility that annotations do not occur during audio time

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#409] [Ideas]
This could happen if eg an annotation file gets misattributed to an audio, or it gets created based on the wrong audio. So this would be in order to catch such mistakes.
This probably happens infrequently, so let's not plan on adding this feature unless other people start asking for it.

**Comment by LoannPeurey** [comment]
At the same time, it does look like a very simple feature to add. So if it proves to be really easy to implement, we probably should

**Reply by alecristia** [reply]
let's ponder it for at least 4 weeks!
